app/controllers/api/generic_works_controller.rb

```diff
@@ -0,0 +1,98 @@
+module API
+  # Adds an endpoint that customizes the update action on nested works
+  class GenericWorksController < ApplicationController
```

Comment:
No problem with having an API. My issue is that it's specific to `GenericWork`s.  What if I add an `Image` type of work?